source/io/module_httpClient.js

```diff
@@ -44,8 +44,7 @@
 
         // WebVI Specific
         WEBVI_UNSUPPORTED_INPUT: 363798,
-        INVALID_HEADER: 363798,
-        FORBIDDEN_HEADER_WARNING: 363798
+        INVALID_HEADER: 363798
```

Comment:
the eslint check complained when I had the comma in there.

/home/travis/build/ni/VireoSDK/source/io/module_httpClient.js
   47:31  error  Unexpected trailing comma                comma-dangle

I'm not sure if it would require changing all other structures in the file, or a change in the travis tests to allow trailing commas.